projects/iio_demo/src/app/main.c

```diff
@@ -194,12 +213,48 @@ int main(void)
 		.size = MAX_SIZE_BASE_ADDR
 	};
 
+	struct iio_data_buffer buff = {
+		.buff = (void *)ADC_DDR_BASEADDR,
+		.size = MAX_SIZE_BASE_ADDR
+	};
+
+	struct iio_data_buffer buff1 = {
+		.buff = (void *)DAC_DDR_BASEADDR,
+		.size = MAX_SIZE_BASE_ADDR
+	};
+
+	uint16_t** buf = (uint16_t**)malloc(10*sizeof(uint16_t*));
```

Comment:
in iio_adc_demo.h : #define ADC_NUM_CHANNELS or something
and here: uint16_t loopback_buff[ADC_NUM_CHANNELS][MAX_SAMPLES_PER_CHANNEL]